tests/cli/conftest.py

```diff
@@ -0,0 +1,36 @@
+"""Shared pytest fixtures for cli module."""
+import pytest
+from click.testing import CliRunner
+
+VALID_TEST_CONFIG = """
+base_model: HuggingFaceTB/SmolLM2-135M
+datasets:
+  - path: mhenrichsen/alpaca_2k_test
+    type: alpaca
+sequence_len: 2048
+max_steps: 1
+micro_batch_size: 1
+gradient_accumulation_steps: 1
+learning_rate: 1e-3
+special_tokens:
+  pad_token: <|end_of_text|>
```

Comment:
```suggestion
  pad_token: <|endoftext|>
```
see https://huggingface.co/HuggingFaceTB/SmolLM2-135M/blob/main/tokenizer_config.json#L5